boards/arm/nrf9160_pca10090/nrf9160_pca10090_partition_conf.dts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Default Flash planning for nRF9160_pca10090.
+ *
+ * Zephyr build for nRF9160 with ARM TrustZone-M support,
+ * implies building Secure and Non-Secure Zephyr images.
+ *
+ * Secure image will be placed, by default, in flash0
+ * (or in slot0, if MCUboot is present).
+ * Secure image will use sram0 for system memory.
+ *
+ * Non-Secure image will be placed in slot0_ns, and use
+ * sram0_ns for system memory.
+ *
+ * Note that the Secure image only requires knowledge of
+ * the beginning of the Non-Secure image (not its size).
+ */
+
+&flash0 {
+	reg = <0x00000000 DT_SIZE_K(256)>;

```

Comment:
This is probably wrong, we are partition a space of one megabyte. Could you please double check?

Reply:
Yes, this is deliberate; this is a Secure nrf9160_pca10090 board - and requires to save some space for the non-secure image. If no-MCU boot is present, then the first 256 KB are allocated to the Secure image.

Hopefully we are going to address the problem of partitioning, properly, together with multi-image building

Reply:
This will limit the amount of flash available to secure firmware to 256k, but assumes that a non-secure firmware image is also present, which might note be the case.

Reply:
I agree that this is suitable for now. Future solutions will make this more configurable.

Reply:
Then let's leave this for now and address it properly in the context of multi-image builds.